app/views/articles/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<p style="color: green"><%= notice %></p>
+<p>Article #<%= @article.id %> <p>
```

Comment:
Can we move this string into a locale file? It's a good practice to prepare ourselves for internationalisation (i18n) from the beginning. It also separates our content from our behaviour.